idaes/power_generation/flowsheets/sofc/surrogates/cpu_surrogate_methods.py

```diff
@@ -1,3 +1,16 @@
+##############################################################################
```

Comment:
Same comment: is this file intended to be part of this PR?

Reply:
I had to fix those files to pass the tests